README.md

```diff
@@ -1 +1,33 @@
-# math-magicians
\ No newline at end of file
+# Math Magicians
```

Comment:
- [ ] Just as we have on the template shared earlier, kindly include the prerequisites users need to run the app, like Node, npm and so on